example_config.yml

```diff
@@ -1,9 +1,11 @@
 objective-filter:
   - TV1D[data_fit=quad,delta=0.9,reg=0.5]
   - TV1D[data_fit=huber,delta=0.9,reg=0.5]
-#dataset:
-#  - Simulated Sinusoidal[sigma=0.1,mu=0,K=500,len_output=400,type_A=random_nonsquare]
+dataset:
+  - Simulated blocked[sigma=0.1,mu=0,n_features=50,n_samples=40,type_A=random_nonsquare, num_block=3]
 #solver:
-#  - all
-n-repetitions: 5
-max-runs: 100
+#  - Celer synthesis
+#  - ADMM analysis and synthesis
```

Comment:
why is this commented?

Reply:
Because I want to run all the solvers, but the command ```all``` doesn't work, so I have to deleted all the lines about solvers to run all.